Calibration/HcalAlCaRecoProducers/plugins/AlCaIsoTracksFilter.cc

```diff
@@ -198,141 +198,152 @@ bool AlCaIsoTracksFilter::filter(edm::Event& iEvent, edm::EventSetup const& iSet
 			   << iEvent.bunchCrossing();
 
   //Step1: Find if the event passes one of the chosen triggers
-  trigger::TriggerEvent triggerEvent;
-  edm::Handle<trigger::TriggerEvent> triggerEventHandle;
-  iEvent.getByToken(tok_trigEvt_, triggerEventHandle);
-  if (!triggerEventHandle.isValid()) {
-    edm::LogWarning("HcalIsoTrack") << "Error! Can't get the product "
-				    << triggerEvent_.label() ;
+  bool ok(false);
```

Comment:
@mmusich I shall name it differenly then
________________________________
From: Marco Musich [notifications@github.com]
Sent: 12 February 2016 10:31
To: cms-sw/cmssw
Cc: Sunanda Banerjee
Subject: Re: [cmssw] bsunanda:Run2 alca35 Add IsoTrackFilter for normal physics events (#13251)


In Calibration/HcalAlCaRecoProducers/plugins/AlCaIsoTracksFilter.cc<https://github.com/cms-sw/cmssw/pull/13251#discussion_r52720178>:

> @@ -198,141 +198,152 @@ bool AlCaIsoTracksFilter::filter(edm::Event& iEvent, edm::EventSetup const& iSet
>                          << iEvent.bunchCrossing();
>
>    //Step1: Find if the event passes one of the chosen triggers
> -  trigger::TriggerEvent triggerEvent;
> -  edm::Handle<trigger::TriggerEvent> triggerEventHandle;
> -  iEvent.getByToken(tok_trigEvt_, triggerEventHandle);
> -  if (!triggerEventHandle.isValid()) {
> -    edm::LogWarning("HcalIsoTrack") << "Error! Can't get the product "
> -                                 << triggerEvent_.label() ;
> +  bool ok(false);


@bsunanda<https://github.com/bsunanda> for the sake of code readibilty would it be possible to find a variable name more meaningful than "ok"?

—
Reply to this email directly or view it on GitHub<https://github.com/cms-sw/cmssw/pull/13251/files#r52720178>.